examples/k8petstore/k8petstore.sh

```diff
@@ -46,173 +46,193 @@ function create {
-          "hostPort": 6379
-        }]
-      }]
+  "apiVersion": "v1beta3",
+  "metadata": {
+    "name": "redismaster",
+    "labels": {
+      "name": "redis-master"
     }
   },
-  "labels": {
-    "name": "redis-master"
+  "spec": {
+    "containers": [{
+      "name": "master",
+      "image": "jayunit100/k8-petstore-redis-master:$VERSION",
+      "ports": [{
+        "containerPort": 6379,
+        "hostPort": 6379
+      }]
+    }]
   }
 }
 EOF
 
 cat << EOF > rm-s.json
 {
-  "id": "redismaster",
   "kind": "Service",
-  "apiVersion": "v1beta1",
-  "port": 6379,
-  "containerPort": 6379,
-  "selector": {
-    "name": "redis-master"
+  "apiVersion": "v1beta3",
+  "metadata": {
+    "name": "redismaster",
+    "labels": {
+      "name": "redis-master"
+    }
   },
-  "labels": {
-    "name": "redis-master"
+  "spec": {
+    "ports": [{
+      "port": 6379,
+      "targetPort": 6379
+    }],
+    "selector": {
+      "name": "redis-master"
+    }
   }
 }
 EOF
 
 cat << EOF > rs-s.json
 {
-  "id": "redisslave",
   "kind": "Service",
-  "apiVersion": "v1beta1",
-  "port": 6379,
-  "containerPort": 6379,
-  "labels": {
-    "name": "redisslave"
+  "apiVersion": "v1beta3",
+  "metadata": {
+    "name": "redisslave",
+    "labels": {
+      "name": "redisslave"
+    }
   },
-  "selector": {
-    "name": "redisslave"
+  "spec": {
+    "ports": [{
+      "port": 6379,
+      "targetPort": 6379
+    }],
+    "selector": {
+      "name": "redisslave"
+    }
   }
 }
 EOF
 
 cat << EOF > slave-rc.json
 {
-  "id": "redissc",
   "kind": "ReplicationController",
-  "apiVersion": "v1beta1",
-  "desiredState": {
+  "apiVersion": "v1beta3",
+  "metadata": {
+    "name": "redissc",
+    "labels": {"name": "redisslave"}
+  },
+  "spec": {
     "replicas": $SLAVE,
-    "replicaSelector": {"name": "redisslave"},
-    "podTemplate": {
-      "desiredState": {
-         "manifest": {
-           "version": "v1beta1",
-           "id": "redissc",
-           "containers": [{
-             "name": "slave",
-             "image": "jayunit100/k8-petstore-redis-slave:$VERSION",
-             "ports": [{"containerPort": 6379, "hostPort": 6380}]
-           }]
-         }
+    "selector": {"name": "redisslave"},
+    "template": {
+      "metadata": {
+        "name": "redissc",
+        "labels": {
+          "name": "redisslave",
+          "uses": "redis-master"
+        }
       },
-      "labels": {
-        "name": "redisslave",
-        "uses": "redis-master"
+      "spec": {
+         "containers": [{
+           "name": "slave",
+           "image": "jayunit100/k8-petstore-redis-slave:$VERSION",
+           "ports": [{"containerPort": 6379, "hostPort": 6380}]
+         }]
       }
     }
-  },
-  "labels": {"name": "redisslave"}
+  }
 }
 EOF
-$kubectl create -f rm.json --api-version=v1beta1 --namespace=$NS
-$kubectl create -f rm-s.json --api-version=v1beta1 --namespace=$NS
+$kubectl create -f rm.json --api-version=v1beta3 --namespace=$NS
```

Comment:
FYI, you don't need to specify the api version here, since it's specified in the json.